sideboard/config.py

```diff
@@ -27,8 +87,10 @@ def get_config_root():
     return config_root
 
 
-def get_dirnames(pyname, plugin):
+def get_module_and_root_dirs(requesting_file_path, is_plugin):
     """
```

Comment:
Function name changed just for clarity. Feel free to suggest a better name, or to object to renaming this function in general!